src/network/providers/ethereum.js

```diff
@@ -540,8 +540,32 @@ ethereum.getKeyValue = async (
     identity,
     key,
     version = 'latest',
-    versionSearchStrategy = 'exact'
+    versionSearchStrategy = 'exact',
+    followCopyFromIkv = false
 ) => {
+    // Process @@copy_from_ikv instruction if followCopyFromIkv is set to true
+    if (followCopyFromIkv) {
+        // self invoke to get the value first but without redirection
+        const value = await ethereum.getKeyValue(identity, key, version, versionSearchStrategy, false);
+
+        const copyFromIkv_prolog = '@@copy_from_ikv=';
+        if (! value.startsWith(copyFromIkv_prolog)) {
+            return value; // no redirection
+        } else {
+            // the format is:
+            // @@copy_from_ikv=<identity>:<key_in_ikv>
+            const withoutPrefix = rootDirId.substring(copyFromIkv_prolog.length);
```

Comment:
Where is this `rootDirId` coming from? I cannot find it defined. I think this should be the `value` we get above in line 549.